src/app/store/cart/cart.component.scss

```diff
@@ -0,0 +1,104 @@
+.cart-products-head,
+.cart-products,
+.cart-products-subtotal,
+.checkout {
+	width: 75%;
+	// margin: 0px 20px;
+	font-size: 21px;
+  color: black;
+  justify-content: center;
+  align-content: center;
+}
+
+.cart-products-subtotal {
+	display: inline-block;
+	margin: 20px 20px 50px 20px;
+}
+
+.title {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin: 20px 0;
+}
+
+.cart-products-head,
+.cart-products {
+	width: 90%;
+	margin: 0px 20px;
+	display: inline-flex;
+	border-bottom: 1px solid #736d6d;
+	font-size: 18px;
+	color: black;
+}
+
+.cartProduct-head {
+  padding-bottom: 10px;
+}
+
+.cartProduct-name {
+  margin-right: 5px;
+}
+
+.cart-products div {
+	display: flex;
```

Comment:
There's a lot of flex CSS in here. We should be able to just use the built in flex layout module stuff to do this. i.e. fxLayout="row", fxFlex, etc

Reply:
We should try to minimize custom CSS in our components.